### DISCUSSION TITLE
Bringing in records with incorrect dates.

### DISCUSSION THREAD

**Discussion by danielgrando** [prisma/prisma#19460] [Q&A]
Hello everyone, everything good ?

My tables are configured like:

 `createdAt        DateTime           @default(now()) @db.Timestamptz()`

and the records are registered correctly in the DB.

But when I bring the records the dates are changed, I imagine it would be something from my gmt, is -3.

Can I configure this within the ORM or do I need to transform within the code?

Thanks!

**Comment by ludralph** [comment]
Hi @danielgrando 👋 

Can you please elaborate on what you mean by  `when i bring the records, the dates are changed`?

**Reply by ludralph** [reply]
Hi @danielgrando 

It would be really helpful if you could provide the additional information that we had asked about last time. If your problem is resolved by now or somehow doesn't feel relevant any more, you can simply close the Discussion as well.

**Reply by danielgrando** [reply]
Hi  @ludralph 👋, how are you ? sorry for the delay in replying, So I am saving the records in my database with GMT -3,
they thus reflect the reality of the local time, however when I bring these records the javascript ignores the gmt and back +3 hours. I would like to know, please, if there is any possibility of automating this conversion with Prisma, or should I do it by hand?

Thank you very much!

**Reply by nurul3101** [reply]
The issue is likely due to the fact that the `DateTime` values are being stored in the database in a different time zone (probably UTC), and you are seeing these values in your local time zone. Are the dates stored in the UTC time zone or your local time zone?

On a side note, It is a common practice to store `DateTime` values in UTC in the database. This helps to avoid issues with Daylight Saving Time and time zone differences. When you retrieve these values in your application, you can convert them to your local time zone.

**Reply by nurul3101** [reply]
@danielgrando It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
